### ISSUE TITLE
Pro Mahjong Kiwame 64: crashes after title screen

### ISSUE THREAD

**Issue by olivieryuyu** [mupen64plus/mupen64plus-core#516]
common in many emulators

1964 overcomes the issue I think

**Comment by loganmc10** [comment]
Can confirm, the emulator prints out this error non-stop:

```
Core Warning: Unaligned dword write 3a5d2171
```

I found a video from August of someone playing it on mupen64plus (https://www.youtube.com/watch?v=C_ZqBNli_mY), so I assume this is a regression, I'll try and track down the bad commit

**Comment by olivieryuyu** [comment]
yes but even on this video you have artifacts when arriving at the point where it crashes. Potentially the game was badly programmed and does something unexpected.

**Comment by bsmiles32** [comment]
Confirmed also on my end, but with different ouput (cached_inter, dynarec)
```
Core Warning: Unaligned dword write 190f21b3
```

Pure interp gives a different output:
```
Core Error: reserved opcode: 80098568:C30401
```

**Comment by loganmc10** [comment]
@olivieryuyu could this be related to the GFX plugin? Does this work in Project64? I tested GLideN64 and Angrylion and the issue happens. If I try Glide64mk2, then it looks like it does in that video (I assume that is the GFX plugin they were using).

Basically if I use glide64mk2 or rice, they both work, but lots of GFX issues. If I use GLideN64 or Angrylion, it freezes at that point.

**Comment by olivieryuyu** [comment]
I cannot say impossible but it is highly doubtful that all plugins, included Angrylion, has an issue here.

**Comment by loganmc10** [comment]
@olivieryuyu I just tested 1964 and Project64 with GLideN64 Public Release 3. They have the exact same issue.

Project64 2.3.2 + the bundled version of Glide64 seemed to work without any flickering or other graphical issues. The video plugin bundled with 1964 looked similar to that video (flickering messed up graphics).


So I would maybe look at Project64 + Glide64 for a good reference point. It seems to be related to the GFX plugin. It is strange that Angrylion has the same issue as GLideN64, but we can't just assume that because it's Angrylion that it must be perfect.

**Comment by olivieryuyu** [comment]
ok let me see that and i come back to u

**Comment by olivieryuyu** [comment]
i checked and the game is stuck in the RSP in an infinite loop. LLE plugin cannot bypass this issue. HLE can ignore those commands but then something will go wrong.

Something is not as it should and it is either a RSP or a core issue.
